.circleci/config.yml

```
@@ -76,7 +76,8 @@ commands:
               # If the image parameter was originally quoted, we need to close the quote
               sed -r -i "s@\"(.*)/apollo-ci:${tag} # TODO@\"\1/apollo-ci:${tag}\" # TODO@g" .circleci/config.yml
 
-              if [[ "$flavor" == "stackrox-build" ]] && [[ "<< parameters.repo >>" == "stackrox" ]]; then
+              if [[ "$flavor" == "stackrox-build" ]] && [[ "<< parameters.repo >>" == "stackrox" ||
+                    "$flavor" == "scanner-build" ]] && [[ "<< parameters.repo >>" == "scanner" ]]; then
```

There is something wrong here - this cannot be true.

Did you mean this maybe?

```suggestion
              if [[ ("$flavor" == "stackrox-build" && "<< parameters.repo >>" == "stackrox") ||
                    ("$flavor" == "scanner-build" && "<< parameters.repo >>" == "scanner") ]]; then
```